gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Settings.java

```diff
@@ -330,46 +301,10 @@ static class ConnectionPoolSettings {
         public boolean sslSkipCertValidation = false;
 
         /**
-         * The minimum size of a connection pool for a {@link Host}. By default this is set to 2.
-         */
-        public int minSize = ConnectionPool.MIN_POOL_SIZE;
-
-        /**
-         * The maximum size of a connection pool for a {@link Host}. By default this is set to 8.
+         * The maximum size of a connection pool for a {@link Host}. By default this is set to 500.
```

Comment:
The default should be 128. The value was changed a few times but it should be 128.

Reply:
Thanks for pointing this out. Will address this in a future PR when I make some documentation changes.

Reply:
Actually I'll just CTR this change in.